kirby/api/context.py

```diff
@@ -0,0 +1,60 @@
+import os
+from smart_getenv import getenv
+import json
+from copy import deepcopy
+
+TYPES = {"str": str, "list": list}
```

Comment:
I get that this is required for serialization/deserialization, but `pickle` would handle this in a more "standard" way

Reply:
Now using pickle